tests/integration/components/high-charts-test.js

```diff
@@ -1,5 +1,7 @@
 import { copy } from 'ember-copy';
-import { moduleForComponent, test } from 'ember-qunit';
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, settled } from '@ember/test-helpers';
 import hbs from 'htmlbars-inline-precompile';
```

Comment:
Since we don't depend on this explicitly anymore, let's change it to:

```suggestion
`import { hbs } from 'ember-cli-htmlbars';`
```

Reply:
can do!